tests/asyncio_test_helpers.py

```diff
@@ -0,0 +1,162 @@
+import logging
+import types
+from asyncio import AbstractEventLoop, transports
+from asyncio.protocols import BaseProtocol, Protocol
+from asyncio.transports import Transport
+from typing import Any, List, Tuple
+
+logger = logging.getLogger(__name__)
+
+
+class TimelessEventLoopWrapper:
+    @property  # type: ignore
+    # -- we have no need to make this read/write
+    # (o.O being able to re-write a __class__ is odd!)
```

Comment:
I'm not following what this means. Please don't put comments between a decorator and a method -- it makes it hard to follow the code flow.

Reply:
Sorry, didn't know where to put the comment in that situation. Suspect it was mostly aimed at me in any case — took it out.